src/backend/api/reviews.js

```diff
@@ -0,0 +1,96 @@
+const { query } = require("express");
+const express = require("express");
+const { max } = require("../database");
+const router = express.Router();
+const knex = require("../database");
+
+router.get('/', async (req, res) => {
+    try {
+        const reviewList = await knex('review').select('*')
+        if (!reviewList.length) {
+            res.status(404).json("There are not available any reviews")
+        } else {
+            res.json(reviewList);
+        }
+    } catch (error) {
+        throw res.status(404).json({ error: "There is error in Page try again" });
+    }
+});
+
+router.get('/:id', async (req, res) => {
+    try {
+        const id = req.params.id
+        const reviewList = await knex.from('review').select('*').where('meal_id', id);
+        if (!reviewList.length) {
+            res.status(404).json("There are not available any reviews")
+        } else {
+            res.json(reviewList);
+        }
+    } catch (error) {
+        throw res.status(404).json({ error: "There is error in Page try again" });
+    }
+})
+router.post('/', async (req, res) => {
+    try {
+        const insertData = await knex('review').insert({
+            title: req.body.title,
+            description: req.body.description,
+            meal_id: req.body.meal_id,
+            stars: req.body.stars,
+            created_date: req.body.created_date,
+        });
+        if (!insertData.length) {
+            res.status(403).json("Data is not inserted in Table")
+        } else {
+            res.status(200).json({ message: "Data inserted in table", key: insertData })
+        }
+    } catch (error) {
+        throw res.status(404).json({ error: "There is error in Page try again" });
+    }
+})
+
+router.get('/:id', async (req, res) => {
```

Comment:
This route is defined twice, meaning this one probably overwrites the one on line 20. I think route on line 20 was probably meant to be `GET /api/meals/:meal_id/reviews`; that one you need to implement in the `meals.js` file.